src/main/java/com/mykaarma/urlshortener/util/UrlServiceUtil.java

```diff
@@ -103,7 +103,7 @@ public long getDurationBetweenTwoDatesInSeconds(Date date1, Date date2) {
 	 */
 	public long getRandomId(int hashLength) throws ShortUrlException {
 		
-		long upperBound = (long)Math.pow(64, hashLength) - 1;
+		long upperBound = (long)Math.pow(DEFAULT_RANDOM_ALPHABET.length(), hashLength) - 1;
```

Comment:
@harshit050499 `DEFAULT_RANDOM_ALPHABET` is just the default, what if the user overrides it with a string of different length? IMO we should use `randomAlphabet` variable instead.